.github/CONTRIBUTING.md

```diff
@@ -26,63 +26,55 @@ Please make sure that your pull request contains unit tests covering what's bein
 
 * All code should be compatible with PHPCS 1.5.6, PHPCS 2.x and PHPCS 3.x.
 * All code should be compatible with PHP 5.3 to PHP nightly.
-* All code should comply with the PHPCompatibility coding standards. The ruleset used by PHPCompatibility is largely based on PSR2 with minor variations and some additional checks for documentation and such.
+* All code should comply with the PHPCompatibility coding standards.
+    The ruleset used by PHPCompatibility is largely based on PSR2 with minor variations and some additional checks for documentation and such.
 
 
 Running the Sniff Tests
 -----------------------
-All the sniffs are fully tested with PHPUnit tests. In order to run the tests
-on the sniffs, the following installation steps are required.
+All the sniffs are fully tested with PHPUnit tests. In order to run the tests on the sniffs, the following installation steps are required.
 
-1. Install the latest release or the `master` branch of [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer.git).
+1. Install PHP CodeSniffer and PHP Compatibility by following the instructions in the Readme for either installing with [Composer](https://github.com/wimg/PHPCompatibility/blob/master/README.md#installation-in-a-composer-project-method-1) or via a [Git Checkout to an arbitrary directory](https://github.com/wimg/PHPCompatibility/blob/master/README.md#installation-via-a-git-check-out-to-an-arbitrary-directory-method-2).
```

Comment:
Make the first link text be "installing with Composer" instead of just "Composer", so it's more descriptive, and not perceived to be a link to getcomposer.org.